src/stylesheets/conversation.less

```diff
@@ -310,3 +310,27 @@
 
     }
 }
+
+.sk-accent-color-light {
```

Comment:
yeah basically. currently they have no use but they'll be there for future stuff.